README.md

```diff
@@ -110,7 +111,7 @@ decrypter.deUpx(updateUpxPath, decryptedPath)
 |livingstone-ru|`LIVINGSTONE`|`ES5zbiHTesxFb+zdkjxiqJ+1dOwyOCv2BzCV7fOYDJxf`|`EFwGaFSherxX1k6Hl/U6TiQhGJOzTPDfsPzHg+z9guln`|`fQ5TLnfKUOHq0f7XXdu9b1FGAckqI576ZBPkZfPg`|
 <!--(strings table end)-->
 
-The models sold in Russia are marked with suffix `-ru`, **some or all of the firmwares for which don't use the same strings with their international versions!**
+The models sold in China/Russia are marked with suffixes `-cn`/`-ru`, **some or all of the firmwares for which don't use the same strings with their international versions!**
```

Comment:
The explanation may be a bit confusing. As far as I known, most (if not all) of models sold Russia have different firmwares with their international versions, but only a few Chinese models has different firmwares with international ones.

Reply:
> The explanation may be a bit confusing. As far as I known, most (if not all) of models sold in Russia have different firmwares with their international versions, but only a few Chinese models has different firmwares with international ones.

Perhaps just tell me what you think sounds best? I'm not familiar with the Boox ecosystem, the way I wrote it is more to reduce clutter.

Reply:
> > The explanation may be a bit confusing. As far as I known, most (if not all) of models sold in Russia have different firmwares with their international versions, but only a few Chinese models has different firmwares with international ones.
> 
> Perhaps just tell me what you think sounds best? I'm not familiar with the Boox ecosystem, the way I wrote it is more to reduce clutter.

```suggestion
A few of models sold in China and most (if not all) of models sold in Russia are marked with suffixes `-cn`/`-ru`, **the firmwares for which don't use the same strings with their international versions!**
```
How about this?